Tests/Fixtures/Controller/UsersController.php

```diff
@@ -58,12 +100,6 @@ public function patchUserAction($slug)
 
  // [PATCH] /users/{slug}
 
-    public function lockUserAction($slug)
```

Comment:
as the lock stuff will be handled by the lock method. The patch thing is simply a fallback: https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Routing/Loader/Reader/RestActionReader.php#L525-L539